### PR TITLE
Upgrade to .NET 6

### PR DESCRIPTION
**Pull request by samsmithnz**
Test to show what an Upgrade to .NET 6 would look like. Achieved with the [dotnet upgrade assistant](https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-overview).

PR is in draft until errors can be resolved.

### CODE REVIEW

File: GAMMA/.upgrade-assistant [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (samsmithnz) [inline GAMMA/.upgrade-assistant:1]: I don't think this file is required - it's just a side effect of the upgrade, and I recommend we delete after review.

File: GAMMA/upgrade-assistant.clef [hunk 2]
```diff
@@ -0,0 +1,346 @@
+{"@t":"2022-01-09T23:00:20.6374067Z","@mt":"Hosting starting","@l":"Debug","EventId":{"Id":1,"Name":"Starting"},"SourceContext":"Microsoft.Extensions.Hosting.Internal.Host"}
```
Comment (samsmithnz) [inline GAMMA/upgrade-assistant.clef]: I don't think this file is required - it's just a side effect of the upgrade, and I recommend we delete after review.

File: GAMMA/packages.config [hunk 3]
```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (samsmithnz) [inline GAMMA/packages.config:1]: These references are now part of CS proj

**Comment by samsmithnz** [comment]
There is one error that I'm not sure about:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3323	Unable to find manifest signing certificate in the certificate store.	GAMMA	C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets	3428	

```

**Comment by NerdyMusician** [comment]
I pulled down a zipped copy of the code, and while I don't see the same error listed about certificate signing when I open the solution, I do see that there was some compatibility warnings about using System.Windows.Interactivity.WPF. Changing it out with Microsoft.Xaml.Behaviors.Wpf seemed to resolve most of these however.

**Comment by NerdyMusician** [comment]
Closing this pull request as I have made my own branch where I went through the .NET 6 upgrade along with changes suggested by another user's submitted issue.

**Pull request closed without merging**